File: app/src/main/java/com/dirror/music/widget/LyricsBackgroundAnimationListener.java
package com.dirror.music.widget;

import android.view.animation.Animation;

/* compiled from: MusicApp */
public class LyricsBackgroundAnimationListener implements Animation.AnimationListener {
    public final /* synthetic */ LyricsBackgroundLayerView lyricsBackgroundLayerView;

    public LyricsBackgroundAnimationListener(LyricsBackgroundLayerView lyricsBackgroundLayerView) {
        this.lyricsBackgroundLayerView = lyricsBackgroundLayerView;
    }

    public void onAnimationEnd(Animation animation) {
        this.lyricsBackgroundLayerView.m = null;
    }

    public void onAnimationRepeat(Animation animation) {
    }

    public void onAnimationStart(Animation animation) {
    }
}

File: app/src/main/java/com/dirror/music/widget/lyricBackground/LyricsBackgroundAnimationListener.kt
package com.dirror.music.widget.lyricBackground

import android.view.animation.Animation
import com.dirror.music.widget.LyricsBackgroundLayerView

class LyricsBackgroundAnimationListener(private val lyricsBackgroundLayerView: LyricsBackgroundLayerView) :
    Animation.AnimationListener {
    override fun onAnimationEnd(animation: Animation) {
        lyricsBackgroundLayerView.m = null
    }

    override fun onAnimationRepeat(animation: Animation) {}
    override fun onAnimationStart(animation: Animation) {}
}